index.js

```diff
@@ -73,16 +89,18 @@
 
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 6", "color: red")
-
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
+  const printNameAndPhones = (users) => {
+    users.map((obj) => console.log(obj.name, `\nCell ${obj.phones.cell}`, `\nOffice: ${obj.phones.office}`));
```

Comment:
Arrow function! Love to see it. There are a few tricky things about arrow functions but I'll have a video to go over them later in the week. Also, `.map` isn't necessary here since we don't care about returning a new array - `forEach` would work as well.